thermo/heating_value.ipynb

```diff
@@ -29,24 +29,43 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    print(f"Using Cantera version: {ct.__version__}")
</pre><p>I don't think that the <code>print</code> statements are needed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
That's true if it's the last line in the cell... this way, even if things are moved around, this will still be shown. Also, the "last line in the cell" thing shows the repr which would surround this text with quotes which is IMO a minor (super minor) annoyance 😄

Reply:
Either way is fine with me. Beyond this minor point, I mainly was curious to see how the review process works on this app … 😄